Upstream patch #30 from git master. Will be present in next release.

growpart: adding LANG=C before the command sfdisk --version

Original bug description:
https://bugs.launchpad.net/cloud-utils/+bug/1928167

---

diff -aur a/bin/growpart b/bin/growpart
--- a/bin/growpart
+++ b/bin/growpart
@@ -230,7 +230,7 @@
 		return 0
 	}
 	# expected output: sfdisk from util-linux 2.25.2
-	out=$(sfdisk --version) ||
+	out=$(LANG=C sfdisk --version) ||
 		{ error "failed to get sfdisk version"; return 1; }
 	set -- $out
 	ver=$4
